app/controllers/ingredients_controller.rb

```diff
@@ -6,7 +6,8 @@ def new
   def create
     @ingredient = Ingredient.new(ingredient_params)
 
-    if @ingredient.save
+    if @ingredient.valid?
```

Comment:
How did it slip into here? :) I guess it came from before. Change it back to "if @ingredient.save"